classes/asserters/phpResource.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace mageekguy\atoum\asserters;
+
+use
+	mageekguy\atoum\asserters
+;
+
+class phpResource extends asserters\variable
+{
+	public function setWith($value)
+	{
+		parent::setWith($value);
+
+		if ($this->analyzer->isResource($this->value) === true)
+		{
+			$this->pass();
+		}
+		else
+		{
+			$this->fail($this->_('%s is not a resource', $this));
+		}
+
+		return $this;
+	}
+
+	public function isOfType($type, $failMessage = null)
```

Comment:
Because we should transform `isPersistentCubridConnection` to `persistent cubric connection`, fine.
But `isDbx_result_object`… crazy.
And `isGdPSFont` will be transformed into `gd ps font`, not `gd PS font`.

:-/